onnxruntime/python/tools/tensorrt/perf/post.py

```diff
@@ -37,107 +45,117 @@ def parse_csv(report_file):
     return memory
 
 def get_latency(latency, model_group):
-    latency_columns = ['Model', \
-                        'CPU fp32 \nmean (ms)', \
-                        'CUDA fp32 \nmean (ms)', \
-                        'TRT EP fp32 \nmean (ms)', \
-                        'Standalone TRT fp32 \nmean (ms)', \
-                        'TRT v CUDA EP fp32 \ngain (mean) (%)', \
-                        'EP v Standalone TRT fp32 \ngain (mean) (%)',     
-                        'CUDA fp16 \nmean (ms)', \
-                        'TRT EP fp16 \nmean (ms)', \
-                        'Standalone TRT fp16 \nmean (ms)', \
-                        'TRT v CUDA EP fp16 \ngain (mean) (%)', \
-                        'EP v Standalone TRT fp16 \ngain (mean) (%)' \
-                        ]
-    latency_db_columns = ['Model', 'CpuFp32', 'CudaEpFp32', 'TrtEpFp32', 'StandaloneFp32', 'TrtGain_CudaFp32', 'EpGain_TrtFp32', \
-                        'CudaEpFp16', 'TrtEpFp16', 'StandaloneFp16', 'TrtGain_CudaFp16', 'EpGain_TrtFp16']
+    latency_columns = [model_title]
+    for provider in provider_list: 
+        latency_columns.append(provider + avg_ending)
+    latency_db_columns = table_headers
     latency = adjust_columns(latency, latency_columns, latency_db_columns, model_group)
     return latency
     
 def get_status(status, model_group):
     status_columns = status.keys()
-    status_db_columns = ['Model', 'CpuFp32', 'CudaEpFp32', 'TrtEpFp32', 'StandaloneFp32', 'CudaEpFp16', 'TrtEpFp16', 'StandaloneFp16']
+    status_db_columns = table_headers
     status = adjust_columns(status, status_columns, status_db_columns, model_group)
     return status
-
-def delete_old_records(engine, table_name):
-
-    # delete using cursor for large table
-    conn = engine.raw_connection()
-    cursor = conn.cursor()
-    delete_query = ('DELETE FROM onnxruntime.' + table_name + ' '
-                    'WHERE UploadTime < DATE_SUB(Now(), INTERVAL 100 DAY);'
-                    )
-    cursor.execute(delete_query)
-    conn.commit()
-    cursor.close()
-    conn.close()
     
-def write_table(engine, table, table_name, trt_version, upload_time):
-    delete_old_records(engine, table_name)
+def write_table(ingest_client, table, table_name, trt_version, upload_time):
     if table.empty:
         return
     table = table.assign(TrtVersion=trt_version) # add TrtVersion
     table = table.assign(UploadTime=upload_time) # add UploadTime
-    table.to_sql(table_name, con=engine, if_exists='append', index=False, chunksize=1)
+    ingestion_props = IngestionProperties(
+      database=database,
+      table=table_name,
+      data_format=DataFormat.CSV,
+      report_level=ReportLevel.FailuresAndSuccesses
+    )
+    # append rows
+    ingest_client.ingest_from_dataframe(table, ingestion_properties=ingestion_props)
+
+def get_time():   
+    date_time = time.strftime(time_string_format)
+    return date_time
+
+def get_cuda_cudnn_version(trt_version): 
```

Comment:
seems like there should be a way to find out the versions of the software from the image (perhaps with the appropriate packaging tool like dpkg for ubuntu or rpm for centos) instead of hardcoding everything.